lib/ctblmono.gi

```diff
@@ -357,6 +364,10 @@ end );
 ##
 #M  TestQuasiPrimitive( <chi> ) . . . . . . . . . . . . . . . for a character
 ##
+##  This works also for reducible <chi>.
+##  Note that a representation affording <chi> maps the centre of <chi>
+##  to scalar matrices.
```

Comment:
What is the relevance of this comment (on the centre) at this point? To me, a comment here is for somebody who wants to call the function without caring what it does... Wouldn't this factoid about the centre be more helpful below, perhaps near the "proof" comment?

Reply:
No.
Somebody who wants to call the function should look at the documentation
in the corresponding `.gd` file.
Comments in the implementation part are intended for somebody who wants to look at the code.